pages/login.vue

```diff
@@ -1,82 +1,40 @@
-<script setup>
+<script lang="ts" setup>
     import pkceChallenge from 'pkce-challenge';
 
     const client_id = "20aa48c2719e42c0be5f3b834942f06d";
     const scopes = [];  // https://developer.spotify.com/documentation/general/guides/authorization/scopes/
```

Comment:
fügst du gleich noch die ausgemachten scopes hinzu?

Reply:
Lieber in einem anderen Ticket dachte ich. Das kann ja auch für einzelne Tickets erweitert werden wenn es nötigist